src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEPropertySymbol.vb

```diff
@@ -610,6 +612,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             End Get
         End Property
 
+        Public Overrides ReadOnly Property IsRequired As Boolean
+            Get
+                If Not _lazyIsRequired.HasValue() Then
+                    _lazyIsRequired = _containingType.ContainingPEModule.Module.HasAttribute(Handle, AttributeDescription.RequiredMemberAttribute).ToThreeState()
```

Comment:
>RequiredMemberAttribute [](http://example.com/codeflow?start=122&length=23)

Can this attribute be applied in VB source?

Reply:
It can, but I'm explicitly ignoring it. If, as a VB user, you want to apply the required member attribute to things yourself, you can, but you are responsible for getting it right and using it correctly within the same compilation.

Reply:
Then we have to support the attributes from source for retargeting.